diff --git a/SPECS/libreport.spec b/SPECS/libreport.spec
index dcea294..04ccfaa 100644
--- a/SPECS/libreport.spec
+++ b/SPECS/libreport.spec
@@ -283,14 +283,14 @@ Requires: libreport-web = %{version}-%{release}
 %description plugin-mantisbt
 Plugin to report bugs into the mantisbt.
 
-%package centos
-Summary: %{name}'s CentOS Bug Tracker workflow
+%package openela
+Summary: %{name}'s OpenELA Bug Tracker workflow
 Requires: %{name} = %{version}-%{release}
 Requires: libreport-web = %{version}-%{release}
 Requires: libreport-plugin-mantisbt = %{version}-%{release}
 
-%description centos
-Workflows to report issues into the CentOS Bug Tracker.
+%description openela
+Workflows to report issues into the OpenELA Bug Tracker.
 
 %package plugin-ureport
 Summary: %{name}'s micro report plugin
@@ -299,10 +299,10 @@ Requires: %{name} = %{version}-%{release}
 Requires: libreport-web = %{version}-%{release}
 %if 0%{?rhel}
 %if %{with python2}
-Requires: python-rhsm
+# Requires: python-rhsm
 %endif # with python2
 %if %{with python3}
-Requires: python3-subscription-manager-rhsm
+# Requires: python3-subscription-manager-rhsm
 %endif # with python3
 %endif
 
@@ -315,14 +315,14 @@ Requires: %{name} = %{version}-%{release}
 Requires: libreport-web = %{version}-%{release}
 
 %description plugin-rhtsupport
-Plugin to report bugs into RH support system.
+Plugin to report bugs into RH support system. This contains no files on OpenELA
+systems.
 
 %if %{with bugzilla}
 %package compat
 Summary: %{name}'s compat layer for obsoleted 'report' package
 Requires: libreport = %{version}-%{release}
 Requires: %{name}-plugin-bugzilla = %{version}-%{release}
-Requires: %{name}-plugin-rhtsupport = %{version}-%{release}
 
 %description compat
 Provides 'report' command-line tool.
@@ -357,6 +357,8 @@ Default configuration for reporting bugs via Red Hat infrastructure
 used to easily configure the reporting process for Red Hat systems. Just
 install this package and you're done.
 
+This should contain no files on a OpenELA system.
+
 %package rhel-bugzilla
 Summary: Default configuration for reporting bugs to Red Hat Bugzilla
 Requires: %{name} = %{version}-%{release}
@@ -368,6 +370,8 @@ Default configuration for reporting bugs to Red Hat Bugzilla used to easily
 configure the reporting process for Red Hat systems. Just install this package
 and you're done.
 
+This should contain no files on a OpenELA system.
+
 %package rhel-anaconda-bugzilla
 Summary: Default configuration for reporting anaconda bugs to Red Hat Bugzilla
 Requires: %{name} = %{version}-%{release}
@@ -377,6 +381,9 @@ Requires: libreport-plugin-bugzilla = %{version}-%{release}
 Default configuration for reporting Anaconda problems to Red Hat Bugzilla used
 to easily configure the reporting process for Red Hat systems. Just install this
 package and you're done.
+
+This should contain no files on a OpenELA system.
+
 %endif
 
 %if %{with bugzilla}
@@ -385,9 +392,9 @@ Summary: Default configuration for reporting anaconda bugs
 Requires: %{name} = %{version}-%{release}
 Requires: libreport-plugin-reportuploader = %{version}-%{release}
 %if 0%{?rhel}
-Requires: libreport-plugin-rhtsupport = %{version}-%{release}
+# Requires: libreport-plugin-rhtsupport = %{version}-%{release}
 %else
-Requires: libreport-plugin-bugzilla = %{version}-%{release}
+# Requires: libreport-plugin-bugzilla = %{version}-%{release}
 %endif
 
 %description anaconda
@@ -464,7 +471,6 @@ rm -f %{buildroot}/%{_datadir}/libreport/workflows/workflow_AnacondaFedora.xml
 %endif
 
 # Remove unwanted RHEL specific workflow configuration files
-%if 0%{!?rhel:1}
 rm -f %{buildroot}/%{_datadir}/libreport/workflows/workflow_RHELCCpp.xml
 rm -f %{buildroot}/%{_datadir}/libreport/workflows/workflow_RHELKerneloops.xml
 rm -f %{buildroot}/%{_datadir}/libreport/workflows/workflow_RHELPython.xml
@@ -496,10 +502,13 @@ rm -f %{buildroot}/%{_sysconfdir}/libreport/workflows.d/report_rhel.conf
 rm -f %{buildroot}/%{_sysconfdir}/libreport/workflows.d/report_rhel_add_data.conf
 rm -f %{buildroot}/%{_sysconfdir}/libreport/workflows.d/report_uReport.conf
 rm -f %{buildroot}/%{_sysconfdir}/libreport/workflows.d/report_rhel_bugzilla.conf
+rm -f %{buildroot}%{_mandir}/man1/reporter-rhtsupport.1
+rm -f %{buildroot}%{_mandir}/man5/rhtsupport.conf.5
+rm -f %{buildroot}%{_mandir}/man5/rhtsupport_event.conf.5
 rm -f %{buildroot}%{_mandir}/man5/report_rhel.conf.5
 rm -f %{buildroot}%{_mandir}/man5/report_uReport.conf.5
 rm -f %{buildroot}%{_mandir}/man5/report_rhel_bugzilla.conf.5
-%endif
+rm -f %{buildroot}/%{_datadir}/dbus-1/interfaces/com.redhat.problems.configuration.rhtsupport.xml
 
 %check
 make check|| {
@@ -724,36 +733,24 @@ gtk-update-icon-cache %{_datadir}/icons/hicolor &>/dev/null || :
 %{_mandir}/man5/mantisbt_format_analyzer_libreport.conf.5.*
 %{_mandir}/man5/mantisbt_formatdup_analyzer_libreport.conf.5.*
 
-%files centos
-%{_datadir}/%{name}/workflows/workflow_CentOSCCpp.xml
-%{_datadir}/%{name}/workflows/workflow_CentOSKerneloops.xml
-%{_datadir}/%{name}/workflows/workflow_CentOSPython.xml
-%{_datadir}/%{name}/workflows/workflow_CentOSPython3.xml
-%{_datadir}/%{name}/workflows/workflow_CentOSVmcore.xml
-%{_datadir}/%{name}/workflows/workflow_CentOSXorg.xml
-%{_datadir}/%{name}/workflows/workflow_CentOSLibreport.xml
-%{_datadir}/%{name}/workflows/workflow_CentOSJava.xml
-%{_datadir}/%{name}/workflows/workflow_CentOSJavaScript.xml
-%config(noreplace) %{_sysconfdir}/libreport/workflows.d/report_centos.conf
-%{_mandir}/man5/report_centos.conf.5.*
-%{_datadir}/%{name}/events/report_CentOSBugTracker.xml
-%config(noreplace) %{_sysconfdir}/libreport/events/report_CentOSBugTracker.conf
-%{_mandir}/man5/report_CentOSBugTracker.conf.5.*
-# report_CentOSBugTracker events are shipped by libreport package
-%config(noreplace) %{_sysconfdir}/libreport/events.d/centos_report_event.conf
-%{_mandir}/man5/centos_report_event.conf.5.gz
-
-%files plugin-rhtsupport
-%config(noreplace) %{_sysconfdir}/libreport/plugins/rhtsupport.conf
-%{_datadir}/%{name}/conf.d/plugins/rhtsupport.conf
-%{_datadir}/%{name}/events/report_RHTSupport.xml
-%{_datadir}/%{name}/events/report_RHTSupport_AddData.xml
-%{_datadir}/dbus-1/interfaces/com.redhat.problems.configuration.rhtsupport.xml
-%config(noreplace) %{_sysconfdir}/libreport/events.d/rhtsupport_event.conf
-%{_mandir}/man1/reporter-rhtsupport.1.gz
-%{_mandir}/man5/rhtsupport.conf.5.*
-%{_mandir}/man5/rhtsupport_event.conf.5.*
-%{_bindir}/reporter-rhtsupport
+%files openela
+%{_datadir}/%{name}/workflows/workflow_OpenELACCpp.xml
+%{_datadir}/%{name}/workflows/workflow_OpenELAKerneloops.xml
+%{_datadir}/%{name}/workflows/workflow_OpenELAPython.xml
+%{_datadir}/%{name}/workflows/workflow_OpenELAPython3.xml
+%{_datadir}/%{name}/workflows/workflow_OpenELAVmcore.xml
+%{_datadir}/%{name}/workflows/workflow_OpenELAXorg.xml
+%{_datadir}/%{name}/workflows/workflow_OpenELALibreport.xml
+%{_datadir}/%{name}/workflows/workflow_OpenELAJava.xml
+%{_datadir}/%{name}/workflows/workflow_OpenELAJavaScript.xml
+%config(noreplace) %{_sysconfdir}/libreport/workflows.d/report_openela.conf
+%{_mandir}/man5/report_openela.conf.5.*
+%{_datadir}/%{name}/events/report_OpenELABugTracker.xml
+%config(noreplace) %{_sysconfdir}/libreport/events/report_OpenELABugTracker.conf
+%{_mandir}/man5/report_OpenELABugTracker.conf.5.*
+# report_OpenELABugTracker events are shipped by libreport package
+%config(noreplace) %{_sysconfdir}/libreport/events.d/openela_report_event.conf
+%{_mandir}/man5/openela_report_event.conf.5.gz
 
 %if %{with bugzilla}
 %files compat
@@ -792,55 +789,11 @@ gtk-update-icon-cache %{_datadir}/icons/hicolor &>/dev/null || :
 %{_mandir}/man5/report_fedora.conf.5.*
 %endif
 
-%if 0%{?rhel}
-%files rhel
-%{_datadir}/%{name}/workflows/workflow_RHELCCpp.xml
-%{_datadir}/%{name}/workflows/workflow_RHELKerneloops.xml
-%{_datadir}/%{name}/workflows/workflow_RHELPython.xml
-%{_datadir}/%{name}/workflows/workflow_RHELvmcore.xml
-%{_datadir}/%{name}/workflows/workflow_RHELxorg.xml
-%{_datadir}/%{name}/workflows/workflow_RHELLibreport.xml
-%{_datadir}/%{name}/workflows/workflow_RHELJava.xml
-%{_datadir}/%{name}/workflows/workflow_RHELJavaScript.xml
-%{_datadir}/%{name}/workflows/workflow_RHELAddDataCCpp.xml
-%{_datadir}/%{name}/workflows/workflow_RHELAddDataJava.xml
-%{_datadir}/%{name}/workflows/workflow_RHELAddDataKerneloops.xml
-%{_datadir}/%{name}/workflows/workflow_RHELAddDataLibreport.xml
-%{_datadir}/%{name}/workflows/workflow_RHELAddDataPython.xml
-%{_datadir}/%{name}/workflows/workflow_RHELAddDatavmcore.xml
-%{_datadir}/%{name}/workflows/workflow_RHELAddDataxorg.xml
-%{_datadir}/%{name}/workflows/workflow_RHELAddDataJavaScript.xml
-%{_datadir}/%{name}/workflows/workflow_uReport.xml
-%config(noreplace) %{_sysconfdir}/libreport/workflows.d/report_rhel.conf
-%config(noreplace) %{_sysconfdir}/libreport/workflows.d/report_rhel_add_data.conf
-%config(noreplace) %{_sysconfdir}/libreport/workflows.d/report_uReport.conf
-%{_mandir}/man5/report_rhel.conf.5.*
-%{_mandir}/man5/report_uReport.conf.5.*
-
-%files rhel-bugzilla
-%{_datadir}/%{name}/workflows/workflow_RHELBugzillaCCpp.xml
-%{_datadir}/%{name}/workflows/workflow_RHELBugzillaKerneloops.xml
-%{_datadir}/%{name}/workflows/workflow_RHELBugzillaPython.xml
-%{_datadir}/%{name}/workflows/workflow_RHELBugzillaVmcore.xml
-%{_datadir}/%{name}/workflows/workflow_RHELBugzillaXorg.xml
-%{_datadir}/%{name}/workflows/workflow_RHELBugzillaLibreport.xml
-%{_datadir}/%{name}/workflows/workflow_RHELBugzillaJava.xml
-%{_datadir}/%{name}/workflows/workflow_RHELBugzillaJavaScript.xml
-%config(noreplace) %{_sysconfdir}/libreport/workflows.d/report_rhel_bugzilla.conf
-%{_mandir}/man5/report_rhel_bugzilla.conf.5.*
-
-%files rhel-anaconda-bugzilla
-%{_datadir}/%{name}/workflows/workflow_AnacondaRHELBugzilla.xml
-%endif
-
 %if %{with bugzilla}
 %files anaconda
 %if 0%{?fedora}
 %{_datadir}/%{name}/workflows/workflow_AnacondaFedora.xml
 %endif
-%if 0%{?rhel}
-%{_datadir}/%{name}/workflows/workflow_AnacondaRHEL.xml
-%endif
 %{_datadir}/%{name}/workflows/workflow_AnacondaUpload.xml
 %config(noreplace) %{_sysconfdir}/libreport/workflows.d/anaconda_event.conf
 %config(noreplace) %{_sysconfdir}/libreport/events.d/bugzilla_anaconda_event.conf
@@ -852,6 +805,14 @@ gtk-update-icon-cache %{_datadir}/icons/hicolor &>/dev/null || :
 %{_mandir}/man5/bugzilla_formatdup_anaconda.conf.5.*
 %endif
 
+%files plugin-rhtsupport
+
+%files rhel
+
+%files rhel-bugzilla
+
+%files rhel-anaconda-bugzilla
+
 %changelog
 * Tue Aug 18 2020 - Ernestas Kulik <ekulik@redhat.com> - 2.9.5-15
 - Add patch for rhbz#1867064
